<!-- Phidgets IK launch file -->

<launch>

  #### Nodelet manager ######################################################

  <node pkg="nodelet" type="nodelet" name="ik_manager"
    args="manager" output="screen" />

  #### IK Driver ############################################################

  <node pkg="nodelet" type="nodelet" name="PhidgetsIKNodelet"
    args="load phidgets_ik/PhidgetsIKNodelet ik_manager"
    output="screen">

    # See README.md for more information on the parameters

    # optional param serial, default is -1
    <!-- <param name="serial" value="123456"/> -->

    # optional param digital_inputs_hub_port, used if connected to a VINT hub
    <!-- <param name="digital_inputs_hub_port" value="0"/> -->

    # optional param digital_inputs_is_hub_port_device, used if connected to a VINT hub
    <!-- <param name="digital_inputs_is_hub_port_device" value="false"/> -->

    # optional param digital_outputs_hub_port, used if connected to a VINT hub
    <!-- <param name="digital_outputs_hub_port" value="0"/> -->

    # optional param digital_outputs_is_hub_port_device, used if connected to a VINT hub
    <!-- <param name="digital_outputs_is_hub_port_device" value="false"/> -->

    # optional param analog_inputs_hub_port, used if connected to a VINT hub
    <!-- <param name="analog_inputs_hub_port" value="0"/> -->

    # optional param analog_inputs_is_hub_port_device, used if connected to a VINT hub
    <!-- <param name="analog_inputs_is_hub_port_device" value="false"/> -->

    # optional param publish_rate, defaults to 0
    <!-- <param name="publish_rate" value="0" /> -->
  </node>

</launch>
